test/integration/fixtures/build-withConfig/tsdx.config.js

```diff
@@ -1,8 +1,14 @@
+/* eslint-disable-next-line no-unused-vars */
```

Comment:
should add this to the example in the README too since they both use `tsdx lint`

Reply:
`tsdx lint` will skip `tsdx.config.js`